wpimath/src/main/native/include/frc/autodiff/Expression.h

```diff
@@ -48,14 +49,17 @@ struct WPILIB_DLLEXPORT Expression {
   // Gradients with respect to each argument
```

Comment:
Do these functions still return the gradient with respect to each argument now that the parent's adjoint is passed in?